packages/web-components/src/radio-group/radio-group.options.ts

```diff
@@ -0,0 +1,16 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * RadioGroupOrientation constants
+ * @public
+ */
+export const RadioGroupOrientation = {
+  horizontal: 'horizontal',
+  vertical: 'vertical',
+} as const;
+
+/**
+ * A RadioGroup's orientation can be either horizontal or vertical
+ * @public
+ */
+export type RadioGroupOrientation = ValuesOf<typeof RadioGroupOrientation>;
```

Comment:
Sorry if I wasn't clear before - this is exported from Fast Foundation, you can just re-export that.

Reply:
should be good now